tools/gulptasks/dist-verify.js

```diff
@@ -6,25 +6,39 @@ const { task } = require('gulp');
 const assert = require('node:assert');
 const { existsSync } = require('node:fs');
 const { join } = require('node:path');
-
 const { message } = require('./lib/log');
 
 const DIST_DIR = './build/dist';
+const HIGHCHARTS_PATHS = [
+    'highcharts/code/es5',
+    'highcharts/code/es-modules',
+    'gantt/code',
+    'highmaps/code',
+    'highstock/code'
+];
+const DASHBOARDS_PATHS = [
+    'dashboards/code/css/dashboards.css',
+    'dashboards/code/gfx/dashboards-icons',
+    'datagrid/js-gzip/css/datagrid.css'
+];
 
 /**
  * Checks the build directory for paths that should exist.
  *
+ * @param  {object} argv
+ *         Command line arguments
+ *
  * @return {Promise<void>}
  *         Promise to keep
  */
-async function verify() {
-    const pathsToVerify = [
-        'highcharts/code/es5',
-        'highcharts/code/es-modules',
-        'gantt/code',
-        'highmaps/code',
-        'highstock/code'
-    ];
+async function distVerify(argv) {
+    let pathsToVerify;
+
+    if (argv.dashboards) {
+        pathsToVerify = DASHBOARDS_PATHS;
+    } else {
+        pathsToVerify = HIGHCHARTS_PATHS;
```

Comment:
I would simplify it and use `pathsToVerify = argv.dashboards ? DASHBOARDS_PATHS : HIGHCHARTS_PATHS;`